build.gradle

```diff
@@ -375,8 +375,13 @@ jacocoTestReport {
     reports {
         xml.enabled true
     }
+
+    doLast {
+        println "Generated JaCoCo test coverage report at $buildDir/reports/jacoco/test/html/index.html"
+    }
 }
 
+project.tasks["sonarqube"].dependsOn "test"
 project.tasks["sonarqube"].dependsOn "jacocoTestReport"
```

Comment:
Could we rename this task to something less specific to the backend, such as "coverage"? Also, if it's an independent task, does it work with rspec as well? If not, and if it only works with unit tests, would it make more sense to be an option for unit testing ala `gradlew test --coverage`?

Reply:
This task comes from sonarqube plugin so we cannot rename it, but we could create a coverage task to alias it.

Reply:
This plugin only works with java, so it can't work with rspec tests (but it might work once we migrate to java integration tests in the future). As @Januson said, I've created a `coverage` alias for it. I couldn't find a way to run it using a flag on the test task, so you'll have to run `gradlew test coverage`

Reply:
Sounds good